### PR TITLE
Small logic improvements

### PR DESCRIPTION
**Pull request by newnon**
Added 4 state parameters
background color
background opacity
title color
title opacity

### CODE REVIEW

**Comment by minggo** [comment]
@newnon
Could you please fix conflict?

@andyque
Could you please take a look?

**Comment by newnon** [comment]
Done

**Comment by minggo** [comment]
Thanks @newnon .

File: tools/cocos2d-console [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit b49fad3fb2dbc396e71a5b1178cf2532309cd174
+Subproject commit fed86b061f5bacffa66149498afdcba180bd0063
```
Comment (minggo) [inline tools/cocos2d-console]: Please don't modify submodules.

**Comment by newnon** [comment]
Sorry it's merge bug

**Comment by newnon** [comment]
everything clear

**Comment by minggo** [comment]
Thanks.

**Pull request closed without merging**